### PR TITLE
Database

### PR DESCRIPTION
**Pull request by lauravikanis**
install mongo db + dotenv,
connect to database
#2

### CODE REVIEW

**Review by domko95 — APPROVED** [review]
LGTM 🚀

File: client/src/api/locations.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export async function getLocation() {
+  const response = await fetch("http://localhost:2601/locations");
```
Comment (phlgr) [inline client/src/api/locations.js:2]: This won't work in production anymore, please use relative paths
```suggestion
  const response = await fetch("/locations");
```

File: client/src/api/locations.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  const lists = await response.json();
+  return lists;
+}
+
+export async function getListsByTour(name) {
+  const response = await fetch(`http://localhost:2601/api/locations/${name}`);
```
Comment (phlgr) [inline client/src/api/locations.js:8]: Same as above

File: server.js [hunk 3]
```diff
@@ -11,11 +17,31 @@ app.use(
   express.static(path.join(__dirname, "client/storybook-static"))
 );
 
-// Handle React routing, return all requests to React app
-app.get("*", (req, res) => {
+app.get("/api/locations/:tour", async (req, res) => {
+  const { tour } = req.params;
+  try {
+    const locationValue = await getLocatioNamebyTour(tour);
```
Comment (phlgr) [inline server.js:23]:
```suggestion
    const locationValue = await getLocationNamebyTour(tour);
```

File: lib/locations.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const { collection } = require("./database");
+
+async function getLocatioNamebyTour(tourName) {
```
Comment (phlgr) [inline lib/locations.js:3]:
```suggestion
async function getLocationNamebyTour(tourName) {
```

File: client/src/pages/Tour.js [hunk 5]
```diff
@@ -24,16 +33,9 @@ const Tour = () => {
...
-          <li>Standort</li>
-        </Link>
-        <Link to="/location">
-          <li>Standort</li>
-        </Link>
-        <Link to="/location">
-          <li>Standort</li>
+          {/* {list.items.map((item) => (
+            <li key={item}>{item}</li>
+          ))} */}
```
Comment (phlgr) [inline client/src/pages/Tour.js:38]: Is this comment necessary?

File: lib/locations.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    .map((location) => location.name)
+    .toArray();
+  if (!locationNamebyTour) {
+    console.log("Error - Das gibts nicht");
+    return null;
+  }
+
+  return locationNamebyTour;
+}
+exports.getLocatioNamebyTour = getLocatioNamebyTour;
```
Comment (phlgr) [inline lib/locations.js:15]:
```suggestion
exports.getLocationNamebyTour = getLocationNamebyTour;
```

**Review by phlgr — CHANGES_REQUESTED** [review]

File: client/src/pages/Tour.js (hunk 5 above)
Comment (lauravikanis) [reply client/src/pages/Tour.js:38]: it just works as a reminder for myself, as the page isn't finished yet

**Review by boldCM — APPROVED** [review]
lgtm 🎉

**Pull request merged**